Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.py

```diff
@@ -357,25 +297,40 @@ def status_check(self, data: Dict[str, Any]) -> Any:
         :return: task status response data.
         :rtype: ``Any``
         """
-        task_id = data.get(TASKID)
-        query_params: Dict[str, Any] = {}
-        response = self.http_request(
-            GET, TASK_DETAIL_ENDPOINT.format(taskId=task_id), params=query_params
+        task_id = data.get(TASKID, EMPTY_STRING)
+        poll = data.get(POLL, TRUE)
+        poll_time_sec = data.get(POLL_TIME_SEC, 0)
```

Comment:
as `poll_time_sec` expected to be a number you should convert it from string to number 
```suggestion
        poll_time_sec = arg_to_number(data.get(POLL_TIME_SEC, 0))
```
please fix it **in all the places** where you expected to get an number

Reply:
corrected.